steps/http/session.go

```diff
@@ -110,6 +110,12 @@ func (s *Session) ConfigureRequestBodyJSONProperties(ctx context.Context, props
 	return nil
 }
 
+// ConfigureRequestBodyJSONText sets a json string as the HTTP request's body .
```

Comment:
Rephrase it to `ConfigureRequestBodyJSONText writes the body in the HTTP request as a JSON from text.`

Reply:
Done